src/main/resources/org/sagebionetworks/web/client/widget/CommaSeparatedValuesParserViewImpl.ui.xml

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE ui:UiBinder SYSTEM "http://dl.google.com/gwt/DTD/xhtml.ent">
+<ui:UiBinder xmlns:ui="urn:ui:com.google.gwt.uibinder"
+             xmlns:g="urn:import:com.google.gwt.user.client.ui"
+             xmlns:b="urn:import:org.gwtbootstrap3.client.ui"
+             xmlns:bh="urn:import:org.gwtbootstrap3.client.ui.html">
+    <bh:Div ui:field="container">
+        <b:TextArea addStyleNames="hide-resize" ui:field="commaSeparatedTextBox"
+                    placeholder="Paste comma or tab delimited values here." visibleLines="5"/>
+        <b:html.Div addStyleNames="text-align-right">
```

Comment:
Try a flexbox layout? (mostly for fun, since this works and is the simplest solution for these text elements)